app/assets/stylesheets/style/not_used/testimonials.scss

```diff
@@ -0,0 +1,133 @@
+// Not used and too long dwessman
+.testimonials {
+  margin-bottom: 10;
+}
+
+.testimonials .testimonial-info {
+  color: $fsek-orange;
+  font-size: 16px;
+  margin-top: 18px;
+  padding: 0 15px;
+}
+
+.testimonials .testimonial-info span {
+  position: relative;
+  top: 3px;
+}
+
+.testimonials .testimonial-info em {
+  color: $gray-text;
+  display: block;
+  font-size: 13px;
+}
+
+.testimonials .testimonial-info img {
+  border-radius: 50% !important;
+  border: solid 1px #ccc;
+  float: left;
+  height: 60;
+  margin-right: 15px;
+  padding: 2px;
+  width: 60;
+}
+
+.testimonials .testimonial-author {
+  overflow: hidden;
+}
+
+.testimonials .carousel-arrow {
+  top: -65px;
+  position: relative;
+}
+
+.testimonials .carousel-arrow i {
+  background: #fafafa;
+  color: $gray-text;
+  font-size: 20;
+  padding: 2px 8px;
+}
+
+.testimonials .carousel-arrow i:hover {
+  background: $fsek-orange;
+  color: #fff;
+}
+
+.testimonials .carousel-control {
+  opacity: 1;
+  position: absolute;
+  text-align: right;
+  text-shadow: none;
+  width: 100%;
+}
+
+.testimonials .carousel-control.left {
+  left: auto;
+  right: 27px;
+}
+
+.testimonials .carousel-control.right {
+  right: 0;
+}
+
+.testimonials.testimonials-v1 .item p {
+  background: #fafafa;
+  font-size: 14px;
+  font-style: italic;
+  padding: 15px;
+  position: relative;
+}
+
+.testimonials.testimonials-v1 .item p:after,
+.testimonials.testimonials-v1 .item p:before {
+  bottom: -20;
+  content: " ";
+  display: block;
+  height: 0;
+  left: 80;
+  position: absolute;
+  width: 0;
+}
+
+.testimonials.testimonials-v1 .item p:after {
+  border-left-style: inset; // FF fixes
+  border-left: 0 solid transparent;
+  border-right-style: inset; // FF fixes
+  border-right: 22px solid transparent;
+  border-top: 22px solid #fafafa;
+}
+
+.testimonials.testimonials-v2 .testimonial-info {
+  padding: 0 20;
+}
+
+.testimonials.testimonials-v2 p {
+  background: #fafafa;
+  font-size: 14px;
+  font-style: italic;
+  padding-bottom: 15px;
+  padding: 15px;
+}
+
+.testimonials.testimonials-v2 .carousel-arrow {
+  top: -55px;
+}
+
+.testimonials.testimonials-v2 .item p:after,
+.testimonials.testimonials-v2 .item p:before {
+  bottom: 45px;
+  content: " ";
+  display: block;
+  height: 0;
+  left: 8%;
+  position: absolute;
+  width: 0;
+}
+
+.testimonials.testimonials-v2 .item p:after {
+  border-left-style: inset; // FF fixes
+  border-left: 25px solid transparent;
+  border-right-style: inset; // FF fixes
+  border-right: 0 solid transparent;
+  border-top: 20 solid #fafafa;
```

Comment:
Color literals like `#fafafa` should only be used in variable declarations; they should be referred to via variable everywhere else.